backend/start/routes.js

```diff
@@ -21,19 +21,24 @@ const Route = use('Route')
 Route.get('/', () => {return { greeting: 'Hello world in JSON' }});
 Route.get('/user/all', 'UserController.list_all');
 Route.get('/user/find/:id', 'UserController.list_by_id');
+Route.get('/list/all', 'ListController.list_all').middleware('auth')
+Route.get('/list/find/:id', 'ListController.list_by_id').middleware('auth')
```

Comment:
Remover e adicionar a rota padrão
```suggestion
Route.resource('list', 'ListController').apiOnly().middleware('auth')
```

Reply:
Corrigido